packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,778 @@
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10xzYOyK9ux6v70g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+
+### IdP Routing Rule Object
+
+| Property    | Type                                                                        | Description                                                                         |
+|-------------|-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------|
+| name        | String                                                                      | Human-readable name for this Rule, configurable during creation or updating.        |
+| id          | String                                                                      | Unique identifier for this Rule (read-only).                                        |
+| type        | String                                                                      | Type of the Rule. For IdP Routing Rule objects, this needs to be `Okta:IdpRouting`. |
+| priority    | Integer                                                                     | Used to determine which Rules take precedence.                                      |
+| conditions  | Array                                                                       | No conditions are supported for this Rule type, so this must be an empty array.     |
+| action      | String                                                                      | Either `ALLOW` or `DENY`. Controls whether the user is allowed to proceed.          |
+| requirement | [IdP Routing Rule Requirement Object](#idp-routing-rule-requirement-object) | Specifies the IdP to use.                                                           |
+| status      | String                                                                      | `ACTIVE`  or  `INACTIVE`.                                                           |
+| default     | Boolean                                                                     | `true` for the first instance of this Rule, which gets created by default.          |
+
+#### IdP Routing Rule Requirement Object
+
+| Property | Type   | Description                    |
+|----------|--------|--------------------------------|
+| idpId    | String | Currently, must be `OKTA`.     |
+| type     | String | Currently, must be `okta_idp`. |
+ 
+### IdP Routing Rule Object Example
+
+```json
+{
+    "name": "Catch-all Rule",
+    "id": "rul10y0LatXn0HP2p0g4",
+    "type": "Okta:IdpRouting",
+    "priority": 2,
```

Comment:
I know I gave you the examples, but can we update them all to have `"priority": 0,`? The reason why I gave you other values is that we have a weird bug that doesn't update the priority numbers for other rules after deletion of a rule.

Reply:
Changed priority to 0 in all examples.